Creatify case study (#8075)
PostHog/posthog.com: contents/customers/creatify.md

---
title: Why Creatify switched from Google Analytics to PostHog
customer: Creatify
logo: ../images/customers/creatify/logo.png
logoDark: ../images/customers/creatify/logo-dark.png
featuredImage: ../images/customers/creatify/featured.png
industries:
    - AI
users:
    - Engineering
    - Leadership
    - Founder
toolsUsed:
    - Web analytics
    - Product analytics
date: 2024-03-19
---

> [Creatify](https://creatify.ai/) is a powerful AI video editor that simplifies the process of quickly creating short video ads for use across any platform. Its capability to rapidly generate marketing video content allows users to test strategies and adopt the data-driven approach that Creatify champions.

Reviewer: ```suggestion
[Creatify](https://creatify.ai/) is a powerful AI video editor that simplifies the process of quickly creating short video ads for any platform. Its capability to rapidly generate marketing videos lets users test strategies and adopt the data-driven approach that Creatify champions.
```

> [Creatify](https://creatify.ai/) is an AI-powered video creation platform that makes it easy for businesses such as Done, Poshmark and Storyblocks to quickly create ads for use across any platform. The ability to rapidly create video content enables users to test strategies and adopt the sort of data-driven approach which Creatify itself champions. 

Author: ```suggestion
[Creatify](https://creatify.ai/) is a powerful AI video editor that simplifies the process of quickly creating short video ads for use across any platform. Its capability to rapidly generate marketing video content allows users to test strategies and adopt the data-driven approach that Creatify champions.
```

> “We officially launched Creatify in early 2024 but we’d been in development mode for a while before that,” said founding CTO Xin Zhou, who met his Creatify co-founders while working at Meta. “We used that time to build the product and test ideas. PostHog was key to that — we’d been using it for about six months before we even launched.”

Author: ```suggestion
“We officially launched Creatify in early 2023 but we’d been in development mode for a while before that,” said founding CTO Xin Zhou, who met his Creatify co-founders while working at Meta. “We used that time to build the product and test ideas. PostHog was key to that — we’d been using it for about three months before we even launched.”
```

> “We officially launched Creatify in early 2024 but we’d been in development mode for a while before that,” said founding CTO Xin Zhou, who met his Creatify co-founders while working at Meta. “We used that time to build the product and test ideas. PostHog was key to that — we’d been using it for about six months before we even launched.”

Reviewer 2: Ehhh...didn't see that, should've been either `Early 2024` or `Dec 2023`. @joethreepwood

> Before switching to PostHog, Creatify had also tried using other platforms, such as Google Analytics and Amplitude. In both cases they found the tools unsuitable for their needs.

Reviewer: ```suggestion
Before switching to PostHog, Creatify had tried using other platforms, such as Google Analytics and Amplitude. They found the tools unsuitable for their needs.
```

> “We started asking other founders what they recommended and they all said PostHog is better,” said Xin. “So, we tried it, and found it pretty simple to set up. Then we found out that PostHog also supports A/B testing. And feature flags. And web analytics!”

Reviewer: ```suggestion
“We started asking other founders what they recommended and they all said PostHog is better,” said Xin. “We tried it and found it pretty simple to set up. Then we found out that PostHog also supports A/B testing. And feature flags. And web analytics.”
```

## Using PostHog for web analytics

> When the PostHog web analytics beta first launched in November 2023, Creatify was among the first organizations to opt-in and start testing it out. As confidence grew, they quickly migrated away from Google Analytics entirely. 

Reviewer: ```suggestion
When the PostHog web analytics beta launched in November 2023, Creatify was among the first organizations to opt-in and start testing it out. As confidence grew, they migrated away from Google Analytics entirely. 
```

> “Monitoring our web analytics is definitely much easier with PostHog,” said Xin. “I'm not an expert on Google Analytics and it can take me a long time to figure some things out with it. PostHog is very intuitive though. I can just immediately find the data I need, whereas in GA4 I don’t even know where to start.”

Reviewer: ```suggestion
“Monitoring our web analytics is definitely much easier with PostHog,” said Xin. “I'm not an expert on Google Analytics and it can take me a long time to figure some things out with it. PostHog is very intuitive, though. I can just immediately find the data I need, whereas in GA4 I don’t even know where to start.”
```

<BorderWrapper>
<Quote
    imageSource="/images/customers/creatify_xin.jpg"
    size="md"
    name="Xin Zhou"
    title="CTO at Creatify"

>     quote={`“I don't think I could ever go back to GA4 now that we've adopted PostHog. Web analytics gives us all the metrics we really care about. It is so much easier to use than GA4!”`}

Reviewer: ```suggestion
    quote={`“I don't think I could ever go back to GA4 now that we've adopted PostHog. Web analytics gives us all the metrics we really care about. It is so much easier to use than GA4.”`}
```

/>
</BorderWrapper>

> The team watched with anticipation over the following months as the beta grew and new features released, often sharing feedback to ensure the dashboard included the metrics they cared most about. On occasions when this wasn’t possible they simply created their own product analytics dashboards to breakdown the data in new ways. 

Reviewer: ```suggestion
The team watched with anticipation over the following months as the beta grew and new features released, often sharing feedback to ensure the dashboard included the metrics they cared about most. When this wasn’t possible they simply created their own product analytics dashboards to break down the data in new ways. 
```

> “I don’t think I could ever go back to GA4 now that we’ve adopted PostHog,” said Xin. “Web analytics gives us all the metrics we really care about, and we’ve set up dashboards for breaking down specific segments when we need to...It’s great that we have all the metrics we care about so readily available.” 

Reviewer: ```suggestion
“I don’t think I could ever go back to GA4 now that we’ve adopted PostHog,” said Xin. “Web analytics gives us all the metrics we really care about, and we’ve set up dashboards for breaking down specific segments when we need to. It’s great that we have all the metrics we care about so readily available.” 
```

## The advantages of an all-in-one data stack

> “We use basically every feature in PostHog, including surveys and A/B testing,” said Xin. “Previously I worked at Meta and Airbnb, where we ran a lot of A/B experiments and would be very data-led. At Creatify these tests are still very important to us.”

Reviewer: ```suggestion
“We use basically every feature in PostHog, including surveys and A/B testing,” said Xin. “Previously I worked at Meta and Airbnb, where we ran a lot of A/B experiments and would be very data-led. At Creatify, these tests are still very important to us.”
```

> As an AI startup, Creatify faces unique challenges in terms of the data the team needs to monitor, however. There are some metrics, such as latency and model performance, which Xin wasn’t able to monitor with Google Analytics or Amplitude — but which [PostHog is starting to support](/templates/ai-analytics).

Reviewer: ```suggestion
As an AI startup, Creatify faces unique challenges in terms of the data the team needs to monitor. There are some metrics, such as latency and model performance, which Xin wasn’t able to monitor with Google Analytics or Amplitude — but which [PostHog is starting to support](/templates/ai-analytics).
```
